NTC-Weather-Forecast-COVID-19/NTC-Weather-Forecast-COVID-19/Controller/Weather Forecast/DetailWeatherViewController.swift

```diff
@@ -76,3 +83,12 @@ extension DetailWeatherViewController: UITableViewDelegate, UITableViewDataSourc
         })
     }
 }
+
+extension DetailWeatherViewController: DetailWeatherDelegate {
+    func sendCheckMark(isMark: Bool, index: Int) {
+        let dataWeather = detailWeather.list[index].convertToDataWeather(city: cityWorld.city)
+        isMark ?
+            WeatherManager.shared.createDataDetailWeatherCD(dataWeather: dataWeather) :
+            WeatherManager.shared.deleteDataDetailWeatherCD(dataWeather: dataWeather)
```

Comment:
lạy chúa ^^ Phức tạp quá cho ra params cũng được mà.

Reply:
dùng toán tử 3 ngôi t thấy đơn giản mà ô, có dài phức tạp đâu :<